### ISSUE TITLE
A11y-friendly KUI

### ISSUE THREAD

**Issue by showeimer** [IBM/kui#1693]
Opening this issue to begin the effort to ensure that KUI is meeting accessibility compliance.  A11y is a mandatory item for many product teams around IBM, and we want to make sure everyone is able to use KUI =).  We should support accessibility for both the desktop and web versions of KUI.

Something that may help is to integrated automated accessibility scans, can see this page:  http://ibm.biz/A11yToolsLandingPage.  This will help cut down on the DAP scan violations.  Will still need to test other things like keyboard navigation, high contrast, etc.

**Comment by starpit** [comment]
i think we are in pretty good shape here. i will close this, and we can open issues for specific features as we find them.
